etcdserver/api/etcdhttp/metrics.go

```diff
@@ -57,7 +58,7 @@ func newHealthHandler(srv *etcdserver.EtcdServer) http.HandlerFunc {
 			return
 		}
 		h := checkHealth(srv)
-		d := []byte(fmt.Sprintf(`{"health": "%v"}`, h.Health))
+		d, _ := json.Marshal(h)
```

Comment:
needs a TODO about removing `{"health" : "true"}` from etcdctl cluster-health in the future; I don't think it can be done in the same minor rev without breaking compat